src/event/v10/attributes.rs

```diff
@@ -121,3 +159,23 @@ impl AttributesConverter for Attributes {
         }
     }
 }
+
+#[test]
+fn iterator_test(){
+    let a = Attributes{
```

Comment:
declare as mut

Reply:
so even if I do declare as mut, when it gets passed to .into_iter(), it will get initialised to 0 each time, and wont move ahead!